app/frontend/src/components/form/form.test.js

```diff
@@ -0,0 +1,37 @@
+import { disableInputs, enableInputs } from './form';
+
+describe('form', () => {
+  document.body.innerHTML = '<input id="test-text-input" type="text"/>';
+  const textField = document.querySelector('#test-text-input');
+  textField.value = '01';
+  const dateFields = Array.from(document.querySelectorAll('input[type="text"]'));
+
+  beforeEach(() => {
+    disableInputs(dateFields);
+  });
+
+  describe('disableInputs', () => {
+    test('removes the value from the input element', () => {
+      expect(document.querySelector('#test-text-input').value).toBe('');
```

Comment:
you should be able to re-use what getElementById is above as it selects a 'live' element unlike querySelector